artifacts/CAMARA_common.yaml

```
@@ -370,6 +370,12 @@ components:
                 status: 422
                 code: DEVICE_NOT_APPLICABLE
                 message: The service is not available for the provided device.
+            GENERIC_422_UNIDENTIFIABLE_DEVICE:
+              description: Service is not available for the provided device
+              value:
+                status: 422
+                code:  UNIDENTIFIABLE_DEVICE

```

Got linter error "too many spaces after colon"

```suggestion
                code: UNIDENTIFIABLE_DEVICE

```